index.html

```diff
@@ -0,0 +1,237 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Gardener</title>
+  <link rel="stylesheet" href="css\style.css">
+</head>
+<body>
+  <header id="header">
+    <div class="container flex-box">
+      <h1 class="logo"><a href=""><img src="images\logo.png" alt="Gardener"></a></h1>
+      <nav>
+        <ul class="flex-box">
+              <li><a href="#home">Home</a></li>
+              <li><a href="#about">About</a></li>
+              <li><a href="#contact">Contact</a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <main>
+    <section class="banner">
+      <div class="container flex-box">
+        <div class="panel-left">
+          <h2>Welcome to <span class="txt-secondary">Gardener</span><br> Landscaping Company.</h2>
+          <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+          Lorem Ipsum has been the industry's standard dummy text ever since the 
+          1500s, when an unknown printer took a gallery of type and scrambled it to make.</p>
+          <a href="contact.html" class="btn btn-primary">Contact Us</a>
+        </div>
+        <div class="panel-right">
+          <img src="images\img-1.jpg" alt="Home">
+        </div>
+      </div> 
+    </section> 
+    <div class="container">
+      <section class="block block-center">
+        <div class="title-area">
+          <span class="txt-secondary">Offer</span>
```

Comment:
Instead of span, you can use the **h** tag. Based on your code you can use h5 tag.